development/image-syncer/external-images.yaml

```
@@ -41,7 +41,7 @@ images:
 - source: "quay.io/prometheus-operator/prometheus-operator:v0.58.0"
 - source: "quay.io/prometheus/alertmanager:v0.24.0"
 - source: "quay.io/prometheus/prometheus:v2.38.0"
-- source: "nats:2.8.4-alpine3.15"
+- source: "nats:2.9.0-alpine"
```

Pin Alpine version, as the current `2.9.0-alpine` version might get force pushed on upstream when new version of Alpine is released
```suggestion
- source: "nats:2.9.0-alpine3.16"
```